cli/go.mod

```diff
@@ -36,5 +36,6 @@ require (
 	golang.org/x/sync v0.0.0-20210220032951-036812b2e83c
 	golang.org/x/term v0.0.0-20201210144234-2321bbc49cbf // indirect
 	gopkg.in/check.v1 v1.0.0-20200227125254-8fa46927fb4f // indirect
+	gopkg.in/yaml.v2 v2.4.0 // indirect
```

Comment:
Any idea where this came from? I don't think you added any new dependencies...

Reply:
This was because of a specified version in the pnpm file. Switching to the v3 removes this.

Reply:
This was because of a specified version in the pnpm file. Switching to the v3 removes this.